planner/cascades/testdata/integration_suite_out.json

```diff
@@ -475,16 +475,32 @@
         "Plan": [
           "Projection_13 8000.00 root test.t2.a, test.t2.b",
           "└─Selection_14 8000.00 root gt(test.t1.a, 2)",
-          "  └─HashRightJoin_16 10000.00 root right outer join, inner:TableReader_19 (REVERSED), equal:[eq(test.t1.a, test.t2.a)]",
```

Comment:
It seems this plan become worse than before one. Could you figure it out?

Reply:
Why is this worse than before? Plannercore also chooses merge join for this query.

Reply:
This case reminds us to implement a tool to check whether there is a performance regression after enabling the cascades planner. 😂

Reply:
> This case reminds us to implement a tool to check whether there is a performance regression after enabling the cascades planner. 😂

Sounds great. I think it's necessary. When the physical plan change, I have no idea which one is better.